templates/reader.html.tt2

```diff
@@ -291,6 +298,7 @@
 
 <div class="absolute-options absolute-right">
     <a class="fa fa-arrow-right fa-2x reading-direction" href="#" title="[% c.lh('Reading Direction') %]"></a>
+    <a class="fa fa-arrows-rotate fa-2x toggle-auto-next-page" href="#" title="[% c.lh('Auto Next Page') %]"></a>
```

Comment:
Not a fan of arrows-rotate for the feature since it feels more like a refresh button...  

Have you considered `fa-stopwatch`?